docs/adr/template.md

```diff
@@ -0,0 +1,73 @@
+# [short title of solved problem and solution]
+
+* Status: [proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)] <!-- optional -->
+* Deciders: [list everyone involved in the decision] <!-- optional -->
+* Date: [YYYY-MM-DD when the decision was last updated] <!-- optional -->
+* Affected modules: [which crate or module is changed]
```

Comment:
```suggestion
* Targeted modules: [which crate or module does this change target]
```

I think this wording is better. Since a change targeting `bdk_chain` may very likely affect `bdk_wallet`. However, just mentioning `bdk_chain` seems more appropriate here.